crypto/fipsmodule/service_indicator/service_indicator.c

```diff
@@ -196,177 +208,187 @@ void HMAC_verify_service_indicator(const EVP_MD *evp_md) {
   }
 }
 
+// Returns 1 if the curve corresponding to the given NID is FIPS approved, and
+// 0 otherwise. Only P-224, P-256, P-384, and P-521 curves are FIPS approved.
+static int is_ec_fips_approved(int curve_nid) {
+
+  switch (curve_nid) {
+    case NID_secp224r1:
+    case NID_X9_62_prime256v1:
+    case NID_secp384r1:
+    case NID_secp521r1:
+      return 1;
+    default:
+      return 0;
+  }
+}
+
+// Updates the service indicator if the elliptic curve contained in |eckey|
+// is FIPS approved, otherwise does nothing.
+void EC_KEY_keygen_verify_service_indicator(EC_KEY *eckey) {
+
```

Comment:
Nit: remove the blank line?

Reply:
I added all the blank lines because to me it's more readable that way. If for you it's better without the blank lines I'll remove them?

Reply:
It's just different from the style in all other files and I would personally prefer the no line after the function signature. Also here, some of the `if` statements have their first line followed by a blank lines and some don't.

Reply:
ok, I removed the blank lines after function signatures and after if statements.